src/components/views/rooms/MessageComposerInput.js

```diff
@@ -523,13 +523,27 @@ export default class MessageComposerInput extends React.Component {
         let sendTextFn = this.client.sendTextMessage;
 
         if (contentText.startsWith('/me')) {
```

Comment:
Actually it would have to be

```
        if (contentText.startsWith('/me') ) {
            contentText = contentText.startsWith("/me ") ? contentText.substring(4) : contentText.substring(3);
            // bit of a hack, but the alternative would be quite complicated
            if (contentHTML) contentHTML = contentHTML.replace(/\/me ?/, '');
            sendHtmlFn = this.client.sendHtmlEmote;
            sendTextFn = this.client.sendEmoteMessage;
        }        if (contentText.startsWith('/me') ) {
            contentText = contentText.startsWith("/me ") ? contentText.substring(4) : contentText.substring(3);
            // bit of a hack, but the alternative would be quite complicated
            if (contentHTML) contentHTML = contentHTML.replace(/\/me ?/, '');
            sendHtmlFn = this.client.sendHtmlEmote;
            sendTextFn = this.client.sendEmoteMessage;
        }
```
as /my needs to add a \'s before the text